app/moves/controllers/list-by-status.js

```diff
@@ -1,16 +1,39 @@
 const presenters = require('../../../common/presenters')
+const i18n = require('../../../config/i18n')
+const filters = require('../../../config/nunjucks/filters')
+
+const tableConfig = [
+  {
+    head: i18n.t('name'),
+    row: data => {
```

Comment:
See https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/pull/437/files#r412253094 for a suggestion on allowing additional properties to be passed to the individual cells.